docs/cloud-native-security/index.rst

```diff
@@ -34,10 +30,10 @@ functions.
 **AWS Security Hub** provides the aggregation point for GuardDuty and IAM
 Access Analyzer findings across multiple AWS accounts, acts as the conduit
 to Rackspace systems, and serves as a single pane of glass for all native
-security services which emit findings. It also provides compliance standard
+security services which emit findings. It also provides security standard
```

Comment:
[question: "emit" is an interesting word choice.  Did you mean "omit" (as in don't provide findings)? Or did you mean emit, as in deliver or provide?  If the latter, I'd recommend using one of those two words instead of emit.]

Reply:
It is the latter. Replaced with "generate".